### ISSUE TITLE
issues with main_resnet50_scratch.py with single GPU

### ISSUE THREAD

**Issue by QingLicsaggie** [facebookresearch/FixRes#18]
I trained resnet50 from scratch but with a single GPU. I couldn't get the results shown. I notice the code is for multiple GPUs, and thus is that the reasons?

**Comment by TouvronHugo** [comment]
Hi @QingLicsaggie ,
This may have a small influence but should not cause too big performance gap. What performance do you get? 
What batch size did you use and which learning rates?
I think if you use only 1 GPU you will have to reimplement our repeated augmentation sampler or not use it at all but in this case you might lose a little bit accuracy.
Best regards,
Hugo
